fix(docs): grammar on `cookbook/dexes/` pages (#2210)
tact-lang/tact: docs/src/content/docs/cookbook/dexes/dedust.mdx

:::caution

  It is important to ensure that contracts are deployed. Sending funds to an inactive contract could result in irretrievable loss.

:::

All kinds of swaps use the `SwapStep{:tact}` and `SwapParams{:tact}` structures:

```tact
/// https://docs.dedust.io/reference/tlb-schemes#swapstep
struct SwapStep {
    // The pool that will perform the swap, e.g., pairs like TON/USDT or USDT/DUST
    poolAddress: Address;

>     // The kind of swap to perform; it can only be 0 as of now

Reviewer: ```suggestion
    // The kind of swap to do: can only be 0 as of now
```
